src/enclave_proc/resource_manager.rs

```diff
@@ -49,7 +54,16 @@ pub type UserMemoryRegion = ne_user_memory_region;
 pub type ImageLoadInfo = ne_image_load_info;
 
 /// The internal data type needed for describing an enclave.
-type UnpackedHandle = (u64, u64, u64, Vec<u32>, u64, u64, EnclaveState);
+type UnpackedHandle = (
+    u64,
+    u64,
+    u64,
+    Vec<u32>,
+    u64,
+    u64,
+    EnclaveState,
+    BTreeMap<String, String>,
```

Comment:
This needs to be somehow backwards compatible.
Think about what happens if:
1. You have an enclave launched.
2. You upgrade you nitro-cli to the new version.
3. You try running nitro-cli describe-enclaves.

Reply:
The measurements would be missing. Should this trigger the PCR computing process? Being launched here and not at the run command, I think this would be the critical describe timeout test as it is entirely serial.

Reply:
> Being launched here and not at the run command, I think this would be the critical describe timeout test as it is entirely serial

I don't follow, what do you mean ?

Reply:
Did I understand well that the measurements will be missing if the nitro-cli is updated after launching the enclave?

Reply:
> I don't follow, what do you mean ?

Regarding the large EIFs, If we are concerned that the thread joining in the normal case would timeout, recalculating the missing PCRs when running the describe command would be a worse scenario.

Reply:
No, my concern is that the process running the old version of the nitro-cli would send the information in the wrong format and the command would fail.

could you please test what happens in the above scenario ?

Reply:
Yes, the command failed, I will solve this.